target_chains/ethereum/entropy_sdk/solidity/IEntropy.sol

```diff
@@ -185,43 +96,15 @@ interface IEntropy is EntropyEvents, EntropyEventsV2 {
         address provider
     ) external view returns (EntropyStructs.ProviderInfo memory info);
 
-    function getProviderInfoV2(
-        address provider
-    ) external view returns (EntropyStructsV2.ProviderInfo memory info);
-
-    function getDefaultProvider() external view returns (address provider);
```

Comment:
note that I moved this function over to IEntropyV2 as well since end users may want to call it